Gopkg.lock

```diff
@@ -2,66 +2,67 @@
 
 
 [[projects]]
+  digest = "1:b5f28f7eda54d2695765c778c951a2ace52c06afb7da1f7e0c7485baa5b02100"
   name = "github.com/beevik/etree"
   packages = ["."]
+  pruneopts = ""
   revision = "9d7e8feddccb4ed1b8afb54e368bd323d2ff652c"
   version = "v1.0.1"
 
 [[projects]]
   branch = "master"
-  name = "github.com/crewjam/saml"
```

Comment:
I think we originally had to do this because it would use version 0.2.0 instead of master.  Looking at https://github.com/crewjam/saml/releases, I think it will still need to be done.